modules/thermal_hydraulics/src/userobjects/ADShaftConnectedCompressor1PhaseUserObject.C

```diff
@@ -185,12 +185,12 @@ ADShaftConnectedCompressor1PhaseUserObject::computeFluxesAndResiduals(const unsi
     const Real x1_spd = _speeds[x1];
     const Real x2_spd = _speeds[x2];
 
-    const Real y1_Rp = _Rp_functions[x1]->value(MetaPhysicL::raw_value(flow_rel_corr), Point());
-    const Real y2_Rp = _Rp_functions[x2]->value(MetaPhysicL::raw_value(flow_rel_corr), Point());
+    const auto y1_Rp = _Rp_functions[x1]->value(flow_rel_corr, ADPoint());
+    const auto y2_Rp = _Rp_functions[x2]->value(flow_rel_corr, ADPoint());
     const ADReal Rp_m = (y2_Rp - y1_Rp) / (x2_spd - x1_spd);
     const ADReal Rp = y1_Rp + (speed_rel_corr - x1_spd) * Rp_m;
-    const ADReal y1_eff = _eff_functions[x1]->value(MetaPhysicL::raw_value(flow_rel_corr), Point());
-    const ADReal y2_eff = _eff_functions[x2]->value(MetaPhysicL::raw_value(flow_rel_corr), Point());
+    const ADReal y1_eff = _eff_functions[x1]->value(flow_rel_corr, ADPoint());
+    const ADReal y2_eff = _eff_functions[x2]->value(flow_rel_corr, ADPoint());
```

Comment:
here we are now getting derivatives that were dropped before

Reply:
Yep, addressing this was one of my commits in https://github.com/idaholab/moose/pull/20217. Glad to see there's a better way now!